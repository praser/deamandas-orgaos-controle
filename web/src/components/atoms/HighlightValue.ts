import styled from 'styled-components';

const HighlightValue = styled.p`
  /* Layout */
  margin: 11.2px 0;

  /* Text */
  font-family: Nunito, -apple-system, system-ui, 'Segoe UI', Roboto,
    'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol', 'Noto Color Emoji';
  font-size: 20px;
  font-weight: 700;
  line-height: 24px;
  text-align: left;
  text-size-adjust: 100%;

  /* Appearance */
  color: rgb(90, 92, 105);
`;

export default HighlightValue;
