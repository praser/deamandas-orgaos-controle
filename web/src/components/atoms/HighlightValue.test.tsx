import React from 'react';
import { render, screen } from '@testing-library/react';
import HighlightValue from './HighlightValue';

describe('HighlightValue component', () => {
  let highlightvalue: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<HighlightValue data-testid={testId}>1000</HighlightValue>);
    highlightvalue = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(highlightvalue).toMatchSnapshot();
  });
});
