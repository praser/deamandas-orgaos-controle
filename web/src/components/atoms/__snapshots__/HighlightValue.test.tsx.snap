// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightValue component is expected to match the snapshot 1`] = `
<p
  class="sc-bdvvaa fGaScV"
  data-testid="testid"
>
  1000
</p>
`;
